lib/utils.js

```diff
@@ -114,9 +120,9 @@ function normalizeBinary (binaryPath, platform, arch) {
 normalizeBinary.paths = {
   "firefox on osx": "/Applications/Firefox.app/Contents/MacOS/firefox-bin",
   "beta on osx": "/Applications/FirefoxBeta.app/Contents/MacOS/firefox-bin",
```

Comment:
Have you checked whether FirefoxBeta without spaces is still accurate?

Reply:
When I download the Firefox Beta for macOS using the link at https://www.mozilla.org/en-US/firefox/channel/desktop/, the name of the downloaded application is simply `Firefox.app`, and the `CFBundleName` it is using is the same as the non-beta release:

```
        <key>CFBundleIdentifier</key>
        <string>org.mozilla.firefox</string>
```

As such, *I chose to left this unchanged just in case anyone is currently relying on the current implementation by manually naming downloaded beta versions `FirefoxBeta.app`* (I'm not familiar enough with the whole breadth of the Firefox build result output to know if there's another source for builds of the beta which may be differently named).

Reply:
This behavior appears to parallel the behavior for Windows, which resolves the 'beta' and 'firefox' aliases to 'Mozilla Firefox', which is explained in a comment for resolving the Windows beta alias.

Happy to make the Mac resolution behave similarly to Windows.

Reply:
Firefox Release and Firefox Beta share the same installation locations on Windows and macOS.
The DeveloperEdition is Firefox Beta with a unique installation location.

And Aurora is defunct and dropped 3 years ago - https://release.mozilla.org/firefox/release/2017/04/17/dawn-project-faq

Reply:
I've aligned the handling of the `beta` alias on macOS to the handling on Windows. It's isolated in a single commit, so easy to ditch if that's undesired.